rustls/src/webpki/anchors.rs

```diff
@@ -103,6 +103,14 @@ impl RootCertStore {
     }
 }
 
+impl FromIterator<TrustAnchor<'static>> for RootCertStore {
+    fn from_iter<T: IntoIterator<Item = TrustAnchor<'static>>>(iter: T) -> Self {
+        let mut store = Self::empty();
+        store.extend(iter);
+        store
```

Comment:
I think this might be more natural as:
```suggestion
        Self {
            roots: iter.into_iter().collect(),
        }
```

Reply:
This would reduce coverage by taking a separate path for `Extend`. To be fair, `Extend` is trivial anyway, so go for it regardless?